Formula/codeclimate.rb

```diff
@@ -1,19 +1,18 @@
-require "formula"
-
 class Codeclimate < Formula
-  CODECLIMATE_VERSION = "0.24.2".freeze
-
+  desc "Code Climate CLI"
   homepage "https://github.com/codeclimate/codeclimate"
-  version CODECLIMATE_VERSION
-
-  url "https://github.com/codeclimate/codeclimate/archive/v#{CODECLIMATE_VERSION}.tar.gz"
-  sha1 "c28ce24aea878ecd8b1bd65cc313206bcd36aece"
+  url "https://github.com/codeclimate/codeclimate/archive/v0.24.2.tar.gz"
+  sha256 "f6ab47d602c8a6ec2d4e9f86df829cc31c3059ad025058c7646499b154242351"
 
   def install
     # Alter PATH to ensure `docker' is available
-    system "env", \
-      "PATH=#{HOMEBREW_PREFIX}/bin:#{ENV["PATH"]}", \
-      "PREFIX=#{prefix}", "make", "install"
+    if Formula["docker"].linked_keg.exist?
```

Comment:
> However, I think the real question is do you want a code climate install via homebrew to require docker via homebrew.

Thanks @ablyler, that's definitely the question and you present a very valid and well-considered argument. However, I think we as a team are leaning the other way.

We're still considering things (and as a Linux-user I don't really have a dog in the fight anyway) so it'll probably be a few days until we come to a decision about this change, should we:

- Merge (again) as-is and require a proper brew-installed `docker` going forward
- Merge (again) but additionally with something like I described above that supports non brew-installed `docker`s provided they're present in some reasonable directory (which should include `/usr/local/bin`).

Thanks again for this change though; it's definitely an improvement no matter which way we decide to go

/cc @GordonDiggs @wfleming